### PR TITLE
Fix issue #533: Not all local branches can merge with its remote-tracking branch by a single git pull command at once

### PR DESCRIPTION
**Pull request by YueLinHo**

### CODE REVIEW

File: book/02-git-basics/sections/remotes.asc [hunk 1]
```diff
@@ -189,7 +189,7 @@ $ git remote show origin
 ----
 
 This command shows which branch is automatically pushed to when you run `git push` while on certain branches.
-It also shows you which remote branches on the server you don't yet have, which remote branches you have that have been removed from the server, and multiple branches that are automatically merged when you run `git pull`.
+It also shows you which remote branches on the server you don't yet have, which remote branches you have that have been removed from the server, and multiple local branches that are able to merge automatically with its remote-tracking branch when you run `git pull`.
```
Comment (ben) [inline book/02-git-basics/sections/remotes.asc]: Let's do:

> able to merge automatically with ***their*** remote-tracking branch

I think that's a bit smoother.

Comment (YueLinHo) [inline book/02-git-basics/sections/remotes.asc]: Thanks for correcting. ^_^

**Comment by YueLinHo** [comment]
Updated with amend.

**Comment by ben** [comment]
*Nice.*

**Comment by YueLinHo** [comment]
Thanks. :smile:

**Pull request merged**